SpotifySearch/SpotifySearch/SpotifyTableViewController.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  SpotifyTableViewController.swift
+//  SpotifySearch
+//
+//  Created by Annie Tung on 10/28/16.
+//  Copyright © 2016 Annie Tung. All rights reserved.
+//
+
+import UIKit
+
+class SpotifyTableViewController: UITableViewController {
+    
+    var kaskade = [Kaskade]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        APIRequestManager.manager.getSpotifyData(apiEndPoint: "https://api.spotify.com/v1/search?q=automatic&type=album&limit=50") { (data: Data?) in
+                if let k = self.validKaskadeData(data: data!) {
+                    self.kaskade = k
+                    print("*****kaskade data has been populated")
+            }
+                DispatchQueue.main.async {
+                    self.tableView.reloadData()
+            }
+        }
+    }
+    
+    func validKaskadeData (data: Data) -> [Kaskade]? {
```

Comment:
This function could be a class function in your Kaskade class. Just to follow MVC properly